terraform/cloudwatch.tf.json.template.py

```diff
@@ -1,34 +1,138 @@
+import json
+
 from azul import (
     config,
+    require,
 )
 from azul.terraform import (
     emit_tf,
 )
 
-emit_tf(None if config.disable_monitoring else {
-    "resource": [
+
+def dashboard_body() -> str:
+    # To minify the template and confirm it is valid JSON before deployment we
+    # parse the template file as JSON and then convert it back to a string.
+    with open(config.cloudwatch_dashboard_template) as f:
+        body = json.load(f)
+    body = json.dumps(body)
+
+    def prod_lambda_name(lambda_name: str) -> str:
+        resource, _, suffix = config.unqualified_resource_name_and_suffix(lambda_name)
+        return config.qualified_resource_name(resource, suffix=suffix, stage='prod')
+
+    lambdas = [
+        config.qualified_resource_name('indexer', suffix='-aggregate_retry'),
+        config.qualified_resource_name('indexer', suffix='-aggregate'),
+        config.qualified_resource_name('indexer', suffix='-contribute_retry'),
+        config.qualified_resource_name('indexer', suffix='-contribute'),
+    ]
+    replacements = {
+        '542754589326': config.aws_account_id,
+        'us-east-1': config.region,
+        'azul-index-prod': config.es_domain,
+        **{
+            prod_lambda_name(lambda_name): lambda_name
+            for lambda_name in lambdas
+        },
+        'azul-notifications_fail-prod': config.notifications_queue_name(fail=True),
+        'azul-notifications_retry-prod': config.notifications_queue_name(retry=True),
+        'azul-notifications-prod': config.notifications_queue_name(),
+        'azul-tallies_fail-prod.fifo': config.tallies_queue_name(fail=True),
+        'azul-tallies_retry-prod.fifo': config.tallies_queue_name(retry=True),
+        'azul-tallies-prod.fifo': config.tallies_queue_name(),
```

Comment:
This is still hard-coding the queue names. Why would we solve the problem only for some of the resource names? As previously discussed, please also use unqualified_resource_name_and_suffix here.

Reply:
In addition to that, there is actually a way to avoid explicitly enumerating the indexer lambdas and queues:

https://github.com/DataBiosphere/azul/blob/0738d7bfebefa5c34e0938dc8c717cf5196f8c09/src/azul/queues.py#L393-L398

After addressing the above comment, please add another fixup that addresses this one.